configs/run_local_server_gh_config.yaml

```diff
@@ -20,11 +20,7 @@ graphhopper:
   # Profiles specifying vehicle and weightings for each mode type.
   custom_model_folder: ./web/test-data
   profiles:
-    - name: gtfs_link_mapper
-      vehicle: car
-      weighting: custom
-      custom_model_file: local_car_custom_model.json
-    - name: car_local
+    - name: car
```

Comment:
this `car` naming limitation is unfortunate (but understandable)...could we introduce a new profile named `car` that doesn't use any custom profiles and leave `car_local` as is? doesn't matter that much in this repo, asking more for the configs in the `model` repo. it feels a little sneaky for the general `car` profile to prefer local roads, especially since that's applied to park-and-ride. would the park and ride routing still be sensible with a default `car` that doesn't have any weighting?

though i suppose routes from the default `car` would get mixed into routes using during actual places runs because of our profile prefixing logic, and maybe we don't want those default routes because they're unrealistic and that's why we introduced profiles 😕. are you viewing this park and ride + profile naming restriction as temporary because core GH will get a fix soon, or are we planning on living with this for a while?

Reply:
You've nailed the tradeoffs here 😁 I was trying to weigh each of these details and thought it was better to deal with the slightly odd naming scheme (commented well) than adding an extra profile that would change the behavior of standard auto routing (because the profile must be called `car` and therefore our prefixing logic would include it, as you mention). I think my hope is that this is a short-term issue and GH core will get a fix soon that we can pick up to address this (non-default profile names with custom profiles used for access/egress modes). cc @michaz 

I also made an assumption here that the local car routing profile is ~ good enough for park-and-ride access/egress routing - at least, it seems like a better option than either the freeway-centric profile or the default profile. @alexeisw , does this sound reasonable to you?

(Luckily, the GTFS link mapper was already using a profile that used the local-roads-preferenced custom model, so we're ok there. We just need to comment well in the model repo configs to make it clear how the `car` profile is special)